src/CodeGen_ARM.cpp

```diff
@@ -885,6 +883,9 @@ void CodeGen_ARM::visit(const Load *op) {
             return;
         }
 
+#if LLVM_VERSION < 37
```

Comment:
FYI: the LLVM pass that enables the intrinsic code generation is only available from 3.7.0 on. 3.6.2 does not have it. 
I'm familiar with what LLVM branches Halide is building against, so I kept the old path too. Let me know if you want me to update the current PR to remove that code, or if you prefer doing this in a later commit.